Billing.API/Controllers/InvoiceController.cs

```diff
@@ -13,19 +14,21 @@ namespace Billing.API.Controllers
     [ApiController]
     public class InvoiceController : ControllerBase
     {
-        private static readonly Regex InvoiceFilenameRegex = new Regex(@"^invoice_\d{4}-\d{2}-\d{2}_(\d+)\.pdf$");
+        private static readonly Regex InvoiceFilenameRegex = new Regex(@"^invoice_([A-Z]{2})_\d{4}-\d{2}-\d{2}_(\d+)\.pdf$");
```

Comment:
You are losing backward compatibility, is this OK?

This could be an alternative:

```suggestion
        private static readonly Regex InvoiceFilenameRegex = new Regex(@"^invoice_([A-Z]{2}_)?\d{4}-\d{2}-\d{2}_(\d+)\.pdf$");
```

Reply:
We don't lose the compatibility because of the filename is generated with the invoices endpoint and when the Admin or User get the invoices the filename has the current format. Let me know if you have any doubts.

Reply:
What happens if somebody shared one of these links?

Reply:
Equaly I will review it

Reply:
> What happens if somebody shared one of these links?

Exactly, for it reason I will update it.

Reply:
Done